tools/accuracy_checker/accuracy_checker/annotation_converters/yolo_labeling_converter.py

```diff
@@ -0,0 +1,86 @@
+"""
+Copyright (c) 2018-2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from .format_converter import BaseFormatConverter, ConverterReturn
+from ..utils import read_txt, check_file_existence
+from ..config import PathField
+from ..representation import DetectionAnnotation
+
+
+class YOLOLabelingConverter(BaseFormatConverter):
+    __provider__ = 'yolo_labeling'
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            'data_dir': PathField(is_directory=True, description='Directory with annotations and images'),
+            'labels_file': PathField(description='Labels file')
+        })
+        return params
+
+    def configure(self):
+        self.data_dir = self.get_value_from_config('data_dir')
+        self.labels_file = self.get_value_from_config('labels_file')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        content_errors = None if not check_content else []
+        image_ann_pairs = []
+
+        annotation_files = self.data_dir.glob('*.txt')
+        for ann_file in annotation_files:
+            image_ann_pairs.append((ann_file.name.replace('txt', 'jpg'), ann_file.name))
+
+        num_iterations = len(image_ann_pairs)
+        annotations = []
+        for idx, (identifier, annotation_file) in enumerate(image_ann_pairs):
+            labels, x_mins, y_mins, x_maxs, y_maxs = self.parse_annotation(annotation_file)
+            annotations.append(DetectionAnnotation(identifier, labels, x_mins, y_mins, x_maxs, y_maxs))
+            if check_content:
+                check_file_existence(self.data_dir / identifier)
+                content_errors.append('{}: does not exist'.format(self.data_dir / identifier))
+            if progress_callback and idx % progress_interval == 0:
+                progress_callback(idx * 100 / num_iterations)
+
+        meta = self.generate_meta()
+
+        return ConverterReturn(annotations, meta, content_errors)
+
+    def parse_annotation(self, annotation_file):
+        labels, x_mins, y_mins, x_maxs, y_maxs = [], [], [], [], []
+        for line in read_txt(self.data_dir / annotation_file):
+            label, x, y, width, height = line.split()
+            x_min, y_min, x_max, y_max = self.convert_from_yolo_to_bbox(float(x), float(y), float(width), float(height))
```

Comment:
I think, this function can be useful for other tasks.
Possibly we can move it to utils and give more general name (e.g. xctr_yctr_w_h_to_x1y1x2y2)

Reply:
Done